workspaces/report-portal/plugins/report-portal/README.md

```diff
@@ -0,0 +1,122 @@
+# Report Portal Plugin
```

Comment:
Would be great to give a small summary of what Report Portal is and a link to that.